### PR TITLE
Add github actions support

### PR DESCRIPTION
**Pull request by harsimranmaan**
travis-ci.org is being shutdown soon

### CODE REVIEW

**Comment by harsimranmaan** [comment]
This is what the CI runs would look like https://github.com/harsimranmaan/go-gitlab/actions/runs/563100633

File: .gitlab/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    name: Test ${{ matrix.go }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go }}
+      - uses: golangci/golangci-lint-action@v2
+        with:
+          version: v1.35.0
+      - run: go test -v
```
Comment (svanharmelen) [inline .gitlab/workflows/ci.yml:21]: Do we need this file? I guess we only need `.github/workflows/ci.yml` right?

**Review by svanharmelen — COMMENTED** [review]
Thanks for this one @harsimranmaan! I was planning to look into this, but didn't find time for it yet. So very nice you made a PR for it 👍🏻

I might update the actions a bit later, but looks good to me (except for the file placed in the `gitlab` directory).

**Comment by harsimranmaan** [comment]
Ha, too much gitlab for me of late ;). I'll send in an update on the morning. Feel free to edit if you like.

File: .gitlab/workflows/ci.yml (hunk 1 above)
Comment (harsimranmaan) [reply .gitlab/workflows/ci.yml:21]: Yes. I should have renoved this : facepalm:

**Review by svanharmelen — APPROVED** [review]
Updated a bit, looks good now 👍🏻

**Pull request merged**